content/beginner/300_windows/calico_windows/install_calico.md

```diff
@@ -8,7 +8,7 @@ Apply the Calico manifest from the [aws/amazon-vpc-cni-k8s GitHub project](https
 
 
 ```
-kubectl apply -f https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/master/config/v1.6/calico.yaml
+kubectl apply -f https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/release-1.6/config/v1.6/calico.yaml
```

Comment:
The version is the same 1.6 version which is already used in this part of the workshop and in the [Network Policies workshop](https://www.eksworkshop.com/beginner/120_network-policies/calico/install_calico/)
I just corrected the link, because current link returns 404 now (/master/ should be changed to /release-1.6/ in the link.)
[Network Policies workshop](https://www.eksworkshop.com/beginner/120_network-policies/calico/install_calico/) already has the correct link.